docs/ref/settings.txt

```diff
@@ -1409,6 +1409,13 @@ If not ``None``, this will be used as the value of the ``SCRIPT_NAME``
 environment variable in any HTTP request. This setting can be used to override
 the server-provided value of ``SCRIPT_NAME``, which may be a rewritten version
 of the preferred value or not supplied at all.
+It is also used by :func:`django.setup()` to set the URL resolver script prefix
```

Comment:
If this is meant as a separate paragraph it needs a blank line above. Otherwise, please flow it with the previous line.